src/main/k/tests/bool.kore

```diff
@@ -0,0 +1,112 @@
+module BOOL

```

Comment:
It would be good if all self-contained definitions would also be tested through the KORE parser, to make sure things stay synchronized;
One suggestion would be that all files here would just be required to be parseable by the KORE parser.

Reply:
I think these tests should be moved to `test/resources`, and we should remove `k/tests` directory. I think it's a good practice to keep all tests in one place.
What do you think, @mattoxb ?

Reply:
Anyway, I suggest we merge it as it is now, and add a new task for reorganizing tests and updating the code appropriately.